server/jobs/api2localdiff/api2localdiff.go

```diff
@@ -1,17 +0,0 @@
-/*
```

Comment:
Why Was this here in the first place? It seems like a place to implement API -> Local Streaming AND diffing?

Reply:
an old (1y+) mistake I think, never had any kind of code